src/components/User/Polls/PollRequests/components/PollRequestView.vue

```diff
@@ -57,62 +57,64 @@
             v-for="item in request.requestDetails.creatorDetails.choices"
             :key="item.number"
           >
-            <span>Answer #{{ item.number }}</span>
+            <span>{{ "poll-request-view.answer" | globalize }}</span>
             <span>
               {{ item.description }}
             </span>
           </li>
         </ul>
 
         <label class="data-caption">
-          Poll details
+          {{ "poll-request-view.poll-details" | globalize }}
         </label>
 
         <ul class="key-value-list">
           <li>
-            <span>Number of choices</span>
+            <span>{{ "poll-request-view.number-of-choices" | globalize }}</span>
             <span>{{ request.requestDetails.numberOfChoices }}</span>
           </li>
 
           <li>
-            <span>Start time</span>
+            <span>{{ "poll-request-view.start-time" | globalize }}</span>
             <date-formatter
               :date="request.requestDetails.startTime"
               format="DD MMM YYYY HH:mm:ss"
             />
           </li>
 
           <li>
-            <span>End time</span>
+            <span>{{ "poll-request-view.end-time" | globalize }}</span>
             <date-formatter
```

Comment:
Remove component 'date-formatter', add a filter for deta formatting how in web-client

Reply:
Use in all place instead of component - filter

Reply:
done